gtk/gtk.go

```diff
@@ -7677,6 +7677,7 @@ func TargetEntryNew(target string, flags TargetFlags, info uint) (*TargetEntry,
 		return nil, nilPtrErr
 	}
 	t := (*TargetEntry)(unsafe.Pointer(c))
+	// FIXME
 	// causes setFinilizer error
```

Comment:
This is likely down to the fact that the `TargetEntry` type just points to `C.GtkTargetEntry` and thus is not managed in Go heap.

Above struct should be:
```go
type TargetEntry struct {
    C.GtkTargetEntry
}
```

Then, SetFinalizer will work.

Reply:
done